### ISSUE TITLE
Demo isn't working

### ISSUE THREAD

**Issue by sampoornavirahm** [cjonasw/smartphone-controlled-3d-web-game#2]
I tried to see the demo, but it dosen't seem to open at all.

**Comment by simonbuehler** [comment]
yes, three.js must be version https://cdnjs.cloudflare.com/ajax/libs/three.js/98/three.min.js and not curent one, could you please update the demo i really would like to try your approach!

**Comment by Zireael07** [comment]
Currently it just says there is nothing at the address. So even more broken than before
